.github/.wordlist.txt

```diff
@@ -792,6 +794,7 @@ machineType
 MacOS
 MacOSX
 MacPorts
+macvlan
```

Comment:
where is this used? Is this a word?

Reply:
That's a docker network type: https://docs.docker.com/network/macvlan/ . We used it initially to have virtual hardware visible on the host network. 
It isn't used anymore as we cut it from this PR but we will revive it sooner or later.
Should we remove it @andy31415 ?

Reply:
Fixed